File: webApp/src/components/UserList/UserUiState.ts
import { JsUser } from 'shared';

export interface UserUiState {
  users: JsUser[];
  isLoading: boolean;
  error: string | null;
}

export const initialUserUiState: UserUiState = {
  users: [],
  isLoading: false,
  error: null,
};

File: iosApp/iosApp/Presentation/State/UserUiState.swift
//
//  UserUiState.swift
//  iosApp
//
//  Created by 성규현 on 2/3/26.
//

import Shared

struct UserUiState {
    var users: [User] = []
    var isLoading: Bool = false
    var error: String? = nil
}
